cmd/argo/lint/lint.go

```diff
@@ -88,21 +88,17 @@ func GetFormatter(fmtr string) (Formatter, error) {
 
 // RunLint lints the specified kinds in the specified files and prints the results to os.Stdout.
 // If linting fails it will exit with status code 1.
-func RunLint(ctx context.Context, client apiclient.Client, files, kinds []string, defaultNs, output string, strict bool) {
+func RunLint(ctx context.Context, client apiclient.Client, kinds []string, output string, offline bool, opts LintOptions) {
 	fmtr, err := GetFormatter(output)
 	errors.CheckError(err)
-
-	clients, err := getLintClients(client, kinds)
-	errors.CheckError(err)
-
-	res, err := Lint(ctx, &LintOptions{
-		ServiceClients:   clients,
-		Files:            files,
-		Strict:           strict,
-		DefaultNamespace: defaultNs,
-		Formatter:        fmtr,
-		Printer:          os.Stdout,
-	})
+	clients := getOfflineLintClients(client)
+	if !offline {
```

Comment:
redundant?

Reply:
Yes, i actually missunderstood part of your suggestion before. I will update and remove `NewOfflineWorkflowServiceClient ` as well.